lib/commands/run.js

```diff
@@ -4,12 +4,14 @@ var log = require('db-migrate-shared').log;
 var optimist = require('optimist');
 var load = require('./');
 var transition = load('transition');
+Promise = require('bluebird');
 
-function run (internals, config) {
+function run (internals, config, callback) {
   var action = internals.argv._.shift();
   var folder = action.split(':');
 
   action = folder[0];
+  var toExecute = null;
```

Comment:
always move definitions to the top of the current scope

Reply:
also do not initialize something with `null` just leave it alone when you want to have it published for the scope but filled later.